.github/workflows/docs.yml

```diff
@@ -158,6 +177,25 @@ jobs:
           publish_dir: doc/_build/html/
           cname: docs.pyvista.org
 
+      - name: Remove non-multi version folder
+        run: rm -rf doc
```

Comment:
😨 ... what's going on here? Again, you definitely should try to use https://github.com/peaceiris/actions-gh-pages to deploy a specific directory and not push the current history

Reply:
Yea, we'll have to clean this up. Assume you're already in a good working state and then proceed.

If I were you, I'd deploy to our [dev-docs](https://dev.pyvista.org/api/index.html) first, get that figured out, and then we can fully replace the documentation. Repo is:
https://github.com/pyvista/pyvista-docs-dev

You guys can play around there, and then once that's working, we can completely overwrite the documentation.